views/js/qtiCreator/widgets/component/minMax/minMax.js

```diff
@@ -47,7 +47,10 @@ define([
             //default help message
             helpMessage: __(
                 'The minimum number of choices that the candidate is required to select to form a valid response.'
-            )
+            ),
+
+            // determines if field can be null
+            canBeNull: false,
```

Comment:
I still don't understand, should it be possible to add a _negative_ number or not? If it was confirmed that _zero_ is minimum, could you please add a comment in the ticket?

[This comment](https://oat-sa.atlassian.net/browse/AUT-1389?focusedCommentId=284097) is wrong: in QTI https://www.imsglobal.org/question/qtiv2p1/imsqti_infov2p1.html#element10065  `lowerBound [0..1]` means that this property may or may not exist. It doesn't mean that minimum value is 0 and maximum value is 1. On the other hand, from [Dragos's comment](https://oat-sa.atlassian.net/browse/AUT-1389?focusedCommentId=284097), negative numbers _should_ be possible. His explanation seems logical.

Reply:
Also, ACs say to enter any `float` number. 

Should we ignore that and continue with integers, or should we just have a _plain text-input_ field instead of _stepper_? Like when you add a score for choice:
![Screenshot_22](https://github.com/user-attachments/assets/0d2fadf7-5ae0-4f51-b644-5f81bd84962f)

Reply:
Floating-point values should be allowed, that's for sure.
I will check with Dragos about the negative values. I don't think we need them, though.